tests/pipelines/test_vote.py

```diff
@@ -258,6 +258,7 @@ def test_valid_block_voting_with_transfer_transactions(monkeypatch, b):
     from bigchaindb_common import crypto, util
     from bigchaindb.models import Transaction
     from bigchaindb.pipelines import vote
+    import time
```

Comment:
This can be imported at the top of the file.

The idea of importing under a test function is to prevent an entire test module to fail because of some failing import, which is only required by a few tests. Hence, importing under the test function will only make the affected tests fail, meanwhile the other tests will pass. This helps identifying the cause of a failing test when an import statement is causing it. The reasoning is explained in http://docs.pylonsproject.org/en/latest/community/testing.html#rule-never-import-the-module-under-test-at-test-module-scope

> Import failures in the module-under-test (MUT) should cause individual test cases to fail: they should never prevent those tests from being run. Depending on the testrunner, import problems may be much harder to distinguish at a glance than normal test failures.